source/Features.rst

```diff
@@ -173,11 +173,11 @@ Click on the **Audit Trail** menu item. Through this, you can navigate to the Au
 
 .. image:: images/Audit2.png
 
-You can see the following audit event deatils in the table view : 
+You can see the following audit event details in the table view : 
 
 .. csv-table::
    :file: AuditTable.csv
-   :widths: 70, 70, 70, 70, 70
+   :widths: 100, 100, 100, 100, 100
```

Comment:
The numbers need to add up to 100. So in this case you should put 20, 20, 20, 20, 20